test/clients/rtm/client.js

```diff
@@ -140,6 +140,27 @@ describe('RTM API Client', function () {
 
   });
 
+  describe('Message Updating', function () {
+
+    it('should call a cb with an err when the RTM is not connected', function (done) {
+      var rtm = new RtmAPIClient('fake-token');
+      rtm.updateMessage({}, function (err, res) {
+        expect(err).to.not.equal(null);
+        expect(res).to.equal(null);
+        done();
+      });
+    });
+
+    it('should call a catch cb with an err when the RTM client is not connected', function (done) {
+      var rtm = new RtmAPIClient('fake-token');
```

Comment:
The only issue with this approach is that it's very hard to test "out of the box". Ideally someone should be able to clone the repo and run tests without having to set up a team. Is it possible to generate some sort of fake test token which returns the correct payload but otherwise does nothing?